client/web/src/marketing/SurveyToast.tsx

```diff
@@ -12,41 +13,57 @@ const HAS_DISMISSED_TOAST_KEY = 'has-dismissed-survey-toast'
 interface SurveyCTAProps {
     className?: string
     score?: number
-    onClick?: (score: number) => void
+    onChange?: (score: number) => void
     openSurveyInNewTab?: boolean
 }
 
-export class SurveyCTA extends React.PureComponent<SurveyCTAProps> {
-    public render(): JSX.Element | null {
-        return (
-            <div className={this.props.className}>
-                {range(0, 11).map(score => {
-                    const pressed = score === this.props.score
-                    return (
-                        /* eslint-disable react/jsx-no-bind */
-                        <Link
-                            key={score}
-                            className={`btn btn-primary toast__rating-btn ${pressed ? 'active' : ''}`}
-                            aria-pressed={pressed || undefined}
-                            onClick={() => this.onClick(score)}
-                            to={`/survey/${score}`}
-                            target={this.props.openSurveyInNewTab ? '_blank' : undefined}
-                        >
-                            {score}
-                        </Link>
-                        /* eslint-enable react/jsx-no-bind */
-                    )
-                })}
-            </div>
-        )
+export const SurveyCTA: React.FunctionComponent<SurveyCTAProps> = props => {
+    const history = useHistory()
+    const [focusedIndex, setFocusedIndex] = useState<number | null>(null)
+
+    const handleFocus = (index: number): void => {
+        setFocusedIndex(index)
+    }
+
+    const handleBlur = (): void => {
+        setFocusedIndex(null)
     }
 
-    private onClick = (score: number): void => {
+    const handleChange = (score: number): void => {
         eventLogger.log('SurveyButtonClicked', { score })
-        if (this.props.onClick) {
-            this.props.onClick(score)
+        history.push(`/survey/${score}`)
+
+        if (props.onChange) {
+            props.onChange(score)
         }
     }
+
+    return (
+        <div aria-label="Survey score radio toggle button group" className={props.className} onBlur={handleBlur}>
```

Comment:
```suggestion
        <div className={props.className} onBlur={handleBlur}>
```

Screen readers should declare that it is a group of radio buttons and I think we can use the label alongside this form field to describe the control